src/components/Calculator.js

```diff
@@ -42,6 +43,7 @@ export default function Calculator() {
 
   return (
     <div className="calculator">
+      <QuoteComponent />
```

Comment:
As per the project requirements, the `Quotes` component should be imported into the main component which is `App.js`, and not in the `Calculator` component. Since the `Quotes` component is totally independent of the `Calculator` component, it would make much more sense if it was in the `App.js` component.